server/db/interface.js

```diff
@@ -0,0 +1,76 @@
+var Sequelize = require('sequelize');
+
+var config = require('./postgres.config.js');
+var url = process.env.DATABASE_URL; //SET THIS UP PROPERLY SOON
+
+var listItem = require('./models/ListItem.js');
+var household = require('./models/Household.js');
+var reckoning = require('./models/Reckoning.js');
+var user = require('./models/User.js');
+
+var dbEnvironment = process.env.NODE_ENV;
+
+var schema;
+var shouldForce;
+
+if (dbEnvironment === 'test') {
+  schema = config.testSchema;
+} else {
+  schema = config.mainSchema;
+}
+
+//not sure what the ssl does yet, will look this up in a minute
+//yell at Naomi if she forgets to get back to this
+var db = new Sequelize(url, {ssl: true, {schema: schema}});
+
+var ListItem = db.define('listItem', listItem.attributes, listItem.options);
+ListItem.schema(schema);
+
+var Household = db.define('household', household.attributes, household.options);
+Household.schema(schema);
+
+var Reckoning = db.define('reckoning', reckoning.attributes, reckoning.options);
+Reckoning.schema(schema);
+
+var User = db.define('user', user.attributes, user.options);
+User.schema(schema);
+
+ListItem.belongsTo(Household);
+Household.hasMany(ListItem);
+
+Reckoning.belongsTo(Household);
+Household.hasMany(Reckoning);
+
+//not sure we need this one, we might be able
+//to get it by querying what we already have?
+ListItem.belongsTo(Reckoning);
```

Comment:
In this case, a join between reckoning and listItems through household would probably be easiest. Sequelize's joins are sometimes a little strange to work with, but we can pass raw query strings really easily with `sequelize.query( )`: http://docs.sequelizejs.com/en/latest/docs/raw-queries/